website/docs/language/settings/backends/configuration.mdx

```diff
@@ -4,19 +4,27 @@ page_title: Backend Configuration - Configuration Language
 
 # Backend Configuration
 
-Each Terraform configuration can specify a backend, which defines where
-[state](/language/state) snapshots are stored.
+A backend defines where Terraform stores its [state](/language/state) data files.
 
-You do not need to configure a backend when using Terraform Cloud because
-Terraform Cloud automatically manages state in the workspaces associated with your configuration. If your configuration includes [a `cloud` block](/language/settings/terraform-cloud), it cannot include a `backend` block.
+Terraform uses persisted state data to keep track of the resources it manages. Most non-trivial Terraform configurations either [integrate with Terraform Cloud](/language/settings/terraform-cloud) or use a backend to store state remotely. This lets multiple people access the state data and work together on that collection of infrastructure resources.
+
+This page describes how to configure a backend by adding the [`backend` block](#using-a-backend-block) to your configuration.
+
+-> **Note:** In Terraform versions before 1.1.0, HashiCorp classified backends as standard or enhanced. The enhanced label differentiated the [`remote` backend](/language/settings/backends/remote), which could both store state and perform Terraform operations. This classification has been removed. Refer to [Using Terraform Cloud](/cli/cloud) for details about storing state, executing remote operations, and using Terraform Cloud directly from Terraform.
```

Comment:
This is perhaps just an irrelevant nit, but I've not seen many examples of us talking about "HashiCorp" in the third person in this sort of documentation before; we've typically said "we" instead. I guess that might feel a bit funny here, but saying tat "HashiCorp" as a whole did anything feels a bit funny too; maybe "The Terraform team" or "this documentation" would be reasonable compromises if "we" isn't right here? :thinking:

Reply:
yeah I went back and forth on this.  I wanted to take it out of passive voice, which it was originally (that avoided this problem entirely, and is why I bet it was written that way). I was thinking about how this note is largely for people who are studying for the HashiCorp certification exam so they're not confused, and that's why I went "HashiCorp". It's not technically wrong, but I agree it's not a convention we typically use. 

You know what? I think I'm going to go back to "we". We use "we" throughout and it's also correct (we, being the teams at HashiCorp who know about such things). Thank you for the thoughtful review, Martin!